core/src/main/java/org/infinispan/configuration/cache/AsyncStoreConfiguration.java

```diff
@@ -68,6 +68,7 @@ public int modificationQueueSize() {
    /**
     * Size of the thread pool whose threads are responsible for applying the modifications.
     */
+   @Deprecated
    public int threadPoolSize() {
```

Comment:
`org.infinispan.configuration.serializing.AbstractStoreSerializer` still serializes the thread pool size.

Reply:
Updated to remove, now have to wait for CI to confirm it didn't break anything :(